src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -471,9 +510,8 @@ class WelcomePage extends Disposable {
 	}
 
 	private createDropDown(): void {
-		const dropdownBtn = document.querySelector('#dropdown_btn') as HTMLElement;
-		const dropdown = document.querySelector('.dropdown') as HTMLInputElement;
-
+		const dropdownBtn = document.querySelector('#dropdown-btn') as HTMLElement;
```

Comment:
Using ids with very generic names like `dropdown-btn` and `dropdown` seems like a recipe for disaster if you're querying the whole document. I'd suggest scoping down the search (which is a good idea anyways to reduce the overhead of having to traverse the entire DOM which shouldn't be necessary)

Reply:
Same goes for a lot of the other queries you're doing.